index.html

```diff
@@ -202,34 +321,35 @@ <h1 class="greet">
               </li>
               <li class="Css_2">
                 <img src="assets/cssLogo.png"><p>Css</p>
-              </li>
+              </li> </div> 
             </ul>
             
             <ul class="framework">
             <li >
               <p>FrameWork</p>
             </li>
-            <li><img src="assets/Unionarrow.png"></li>
+            <li class="arrow"><img src="assets/Unionarrow.png"></li>
             </ul>
             <ul class="skill_txt">
               <li>
                 <p>
                 Skills
                 </p>
               </li>
-              <li><img src="assets/Unionarrow.png"></li>
+              <li class="arrow"><img src="assets/Unionarrow.png"></li>
           </ul>
           </ul>
-      </div>
+        </div>
+      </section>
     </main>
     <footer class="contact-container">
       <div class="contact-bg">
       <div class="contacts">Contact me</div>
       <div class="contacts-txt">
         If you have an application you
-        are intrested in developing, a feature
+        are intrested in developing,<br> a feature
         that you need to build or a project that needs
-        coding . I'd love to help with it 
+        coding .<br> I'd love to help with it 
       </div>
```

Comment:
- [ ] You should stick to the design as much as possible. Please remove the black border from the input field, also don't use the border radius to match the design.
- [ ] Please make your `Get in touch` button at the center of the input field. See the image below for reference
- [ ] Please increase the width of the input field and decrease the height of the input field then you are good to go.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213867139-fe8cf32f-9ec8-4d54-aeab-20174dece8f0.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213867154-f49adde1-15fd-418b-8731-b83690b459cc.jpg)